edi/models/edi_exchange_record_mixin.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 ACSONE SA
+# Copyright 2020 Creu Blanca
+# @author Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class EDIExchangeRecordMixin(models.AbstractModel):
+    """Record that might have related EDI Exchange records
+    """
+
+    _name = "edi.exchange.record.mixin"
```

Comment:
```suggestion
    _name = "edi.exchange.consumer.mixin"
```
I named it like this in the original code as it express better the purpose :wink:

Reply:
Done